tool/src/org/antlr/v4/codegen/Target.java

```diff
@@ -689,6 +689,10 @@ public boolean supportsOverloadedMethods() {
 		return true;
 	}
 
+	public boolean ATNSerializedAsInts() {
```

Comment:
I recommend using the following naming for methods: `isATNSerializedAsInts()`. At least the leading letter should be in lowercase.

Reply:
good suggestion. yes, I had trouble coming up with a name that didn't violate rules or look weird. I'll change.